### ISSUE TITLE
How to confirm that the model has converged? Using YTB data training, the first stage loss basically varies between [0.06, 0,07, 0.08], with no convergence trend

### ISSUE THREAD

**Issue by Bobo-y** [OpenDriveLab/PPGeo#7]

**Comment by Bobo-y** [comment]
I download the pretrained ckpt, the pose decoder's state dict is missing.

**Comment by penghao-wu** [comment]
It is true that the training loss fluctuates but overall it has a convergence trend. This is the training log I got when I ran the codebase.
![飞书20230420-152652](https://user-images.githubusercontent.com/62696862/233500210-496ebf79-7ed3-4318-b079-c5b80e580d85.png)
